.github/workflows/revive-llvm-test.yml

```
@@ -0,0 +1,37 @@
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+    paths:
+      - 'LLVM.lock'
+      - 'crates/llvm-builder/**'
+      - '.github/workflows/revive-llvm-test.yml'
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        runner: [parity-large, macos-14, macos-13]
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install apt dependencies
+        if: matrix.runner == 'parity-large'
+        run: |
+          sudo apt update && sudo apt-get install -y cmake ninja-build curl git libssl-dev pkg-config clang lld musl
+
+      - name: Install macos dependencies
+        if: matrix.runner == 'macos-14' || matrix.runner == 'macos-13'
+        run: |
+          brew install ninja
+
```

I wasn't aware that on the parity runner there is no rust installed at all so we should just remove the additional target

```suggestion

      - uses: actions-rust-lang/setup-rust-toolchain@v1
        with:
          toolchain: stable
          components: rust-src
          rustflags: ""

```